notebooks/byod.py

```diff
@@ -105,30 +104,52 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
+        if filename.endswith(".cram.crai"):
+            # double extension in the crai file, ie "foo.cram.crai"
+            sample = filename[:-10]
+        else:
+            sample = filename[:-5]
+        ext = filename[-4:]
```

Comment:
Assuming extension lengths looks brittle, but this can be handled in a future work.